app/dossierIndicators/dossierIndicators.services.js

```diff
@@ -0,0 +1,160 @@
+//Récupération de tous les indicateurs : phase d'initialisation
```

Comment:
Please translate or remove this comment

Reply:
Done. Removed as it wasn't very informative.